### DISCUSSION TITLE
Customize default difficulty selection

### DISCUSSION THREAD

**Discussion by yamplum** [ppy/osu#21985] [Ideas]
When navigating to a beatmap set in song selection, the game will select a difficulty that's meant to match the player's skill. I find that this feature defaults to difficulties below my preferred level (e.g. I usually play 6-7* beatmaps, whereas the default selection is around 5*): 
![image](https://user-images.githubusercontent.com/118788705/210168443-6989ca2d-aaa8-4f30-a254-73635525fc87.png)

I'm not sure how it's calculated, but I suspect it's based on highest submitted scores? In my case I typically play for fun rather than to score, so my recorded performance does not really reflect my skill level.

It would be nice if I could tune the default selection star rating target, either manually via an option or perhaps by locally monitoring the most commonly played beatmap ratings.

Related: #5172, #8419

**Comment by mk56-spn** [comment]
I mean why not set the star range below the search bar then. That way you only see whatever star ratings you wish to see at any given time. This seems like it wouldn't offer much of oky benefit for the hassle of implementing it

**Reply by yamplum** [reply]
I am using the difficulty range selector. It doesn't address the problem fully because there are still low SR maps I want to play (for example, from single-difficulty sets; full-alt slider maps also tend to be undervalued in their SR).

A little bit of polish goes a long way.

**Comment by LittleEndu** [comment]
There was a proposal to change the difficulty recommendation to better match the star rating people play: https://github.com/ppy/osu-web/issues/9020

**Reply by yamplum** [reply]
That could certainly help to some extent, but I'm not too keen that it's still tied to top scores (in this case via pp). Most of my top plays are from 6-7 years ago, and I've been effectively an offline player on lazer for a long time.

Instead of the game trying to guess my preference I'd rather just be able to tell it what I want.
